### PR TITLE
WIP: Notification Screen Design Changes

### PR DESCRIPTION
**Pull request by BilalBudhani**

### CODE REVIEW

**Comment by blargity** [comment]
@BilalBudhani can you test on Android and let us know if this is the case?

**Comment by BilalBudhani** [comment]
@blargity I will run this on Android and let you know.

File: src/screens/Notifications/NotificationsScreen.js [hunk 1]
```diff
@@ -68,70 +68,89 @@ class NotificationsScreen extends Component {
...
+      );
+    }
+    return <View />;
+  };
 
   render() {
-    const { notifications, loadingNotifications } = this.props;
+    const { notifications, loadingNotifications, getNotifications } = this.props;
     return (
-      <View style={{ flex: 1, backgroundColor: '#FAFAFA' }}>
```
[the remark this answers is not in the document]
Comment (BilalBudhani) [reply src/screens/Notifications/NotificationsScreen.js]: @edencakir I've applied the same styles to `FlatList` component which makes it scrollable.

**Comment by BilalBudhani** [comment]
@blargity sorry It took me some time to reply on this issue. I was setting up Android dev tools on my machine to be able to run it on the platform.

File: src/screens/Notifications/NotificationsScreen.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (edencakir) [reply src/screens/Notifications/NotificationsScreen.js]: Alright great.

**Comment by blargity** [comment]
Hey @BilalBudhani, is this still relevant?

**Pull request merged**